Automattic/wp-calypso
# Stats: Updating details page links placement

#### Proposed Changes

* adding an additional text for the highlight section on the `Insights` page, making the header link more visible
* removing the header link from the components with horizontal bars and moving it to the bottom of the card to unify the navigation

This PR is reflecting conversations from [#70905](https://github.com/Automattic/wp-calypso/pull/70905#issuecomment-1341770162)

#### Testing Instructions



* use the live branch to get to the Stats page
* make sure that all of the components having horizontal bars (in the `Traffic` page) don't have links on the component's headers
* make sure that all of the components with horizontal bars have a link at the bottom of the card, navigating to its details page
* components with 10 items should have a `View all` link, while components with fewer items should have a `View details` link
* highlight components on the `Insights` page should have a `View all annual insights` link next to their title
* all of the components on the `Insights` page shouldn't be affected by the title and bottom link changes made to the `Traffic` page

Before:
![Screenshot 2022-12-09 at 10 32 51 AM](https://user-images.githubusercontent.com/112354940/206572521-03904961-3de7-4d8a-b96c-0eb67ee4db3b.png)
![Screenshot 2022-12-09 at 10 33 47 AM](https://user-images.githubusercontent.com/112354940/206572534-7af2470e-7f31-4f75-9ec2-88d7589942f9.png)

After:
![Screenshot 2022-12-09 at 10 35 40 AM](https://user-images.githubusercontent.com/112354940/206572569-db3e42b0-ca85-48dc-9216-09d4ef8279d3.png)
![Screenshot 2022-12-09 at 10 34 21 AM](https://user-images.githubusercontent.com/112354940/206572601-e8e3386a-2bdd-4219-946d-0e8e7ed1569a.png)

#### Pre-merge Checklist





Fixes #70849.
